service/dap/server.go

```diff
@@ -904,8 +914,8 @@ func cleanExeName(name string) string {
 func (s *Session) onLaunchRequest(request *dap.LaunchRequest) {
 	var err error
 	if s.debugger != nil {
-		s.sendShowUserErrorResponse(request.Request, FailedToLaunch,
-			"Failed to launch", "debugger already started - use remote attach to connect to a server with an active debug session")
+		s.sendShowUserErrorResponse(request.Request, FailedToLaunch, "Failed to launch",
```

Comment:
What's the point of printing the server address here? Clearly the user already knows it.

Reply:
Not if they accidentally used the wrong configuration. This happened to me. I had a dlv-mux server running, which I forgot to take down in some tab. Then to check something else I switched to my "Launch" configuration, forgetting that it had a port set. So instead of having vscode-go start dlv-dap, it connected to my mux server. It took me a moment to make sense of the error message. I thought this tweak could help clarify what happened.